extensions/amp-social-share/1.0/social-share.js

```diff
@@ -194,7 +194,7 @@ function getQueryString(endpoint) {
   let h = endpoint.indexOf('#');
   q = q === -1 ? endpoint.length : q;
   h = h === -1 ? endpoint.length : h;
-  return h < q ? '' : endpoint.substr(q, h - q);
+  return endpoint.substr(q, h - q);
```

Comment:
Nit: We can use `slice` since it uses `endIndex` param instead of `length`:

```suggestion
  return endpoint.slice(q, h);
```

Reply:
Thanks, this is much cleaner, updated!